tests/data_files/Makefile

```diff
@@ -864,6 +864,70 @@ ec_prv.pk8param.pem: ec_prv.pk8param.der
 	$(OPENSSL) pkey -in $< -inform DER -out $@
 all_final += ec_prv.pk8param.pem
 
+ec_prv.sec1.comp.pem: ec_prv.sec1.pem
```

Comment:
Non-blocker: it's generally preferable to have test files in DER format (except when testing PEM-specific features), so they can be used in build without PEM.

Reply:
Good, point, though that's pre-existing, in that the files with uncompressed format are already PEM.